docs/general/oasis-network/why-oasis.md

```diff
@@ -97,58 +91,35 @@ The Oasis Network is a Layer 1, proof-of-stake, decentralized network. It has tw
 
 The Oasis Network’s impressive scalability is achieved through a cutting-edge set of features that provide faster transaction speeds and higher throughput than other networks. The top-tier performance of the network is largely due to its separation of compute and consensus operations into the Consensus Layer and ParaTime Layer. This separation allows multiple ParaTimes to process transactions in parallel, meaning complex workloads processed on one ParaTime won’t slow down faster, simpler transactions on another. Plus, the network’s sophisticated discrepancy detection makes Oasis more efficient than sharding and parachains, requiring a smaller replication factor for the same level of security.
 
-![Scalability](images/paratimes.png)
+![Scalability](../images/architecture/technology_scalability.svg)
```

Comment:
Note: this has different keywords in the image

| Before | After |
| --- | --- |
| ![](https://raw.githubusercontent.com/oasisprotocol/docs/f99925c354db18f914410eb453604523b771a05a/docs/oasis-network-primer/images/paratimes.png) | ![](https://raw.githubusercontent.com/oasisprotocol/docs/f99925c354db18f914410eb453604523b771a05a/docs/general/images/architecture/technology_scalability.svg) |

Reply:
Yeah, that was obsolete.